index.html

```diff
@@ -68,21 +66,367 @@
+                    >
+                        <img src="/assets/images/categories/electronics.svg" alt="Electronics">
+                        <div class="category-label-and-arrow">
+                            <span id="electronics-category-label">Electronics</span>
+                            <img src="/assets/icons/down_arrow_black.svg" alt="Down Arrow">
+                        </div>
+                    </a>
+
+                    <a
+                        href="#"
+                        class="d-flex flex-column justify-content-center align-items-center text-decoration-none text-black"
+                        role="button"
+                        aria-labelledby="home-category-label"
+                    >
+                        <img src="/assets/images/categories/home.svg" alt="Home">
+                        <div class="category-label-and-arrow">
+                            <span id="home-category-label">Home</span>
+                            <img src="/assets/icons/down_arrow_black.svg" alt="Down Arrow">
+                        </div>
+                    </a>
+
+                    <a
+                        href="#"
+                        class="d-flex flex-column justify-content-center align-items-center text-decoration-none text-black"
+                        role="button"
+                        aria-labelledby="appliances-category-label"
+                    >
+                        <img src="/assets/images/categories/appliances.svg" alt="Appliances">
+                        <div class="category-label-and-arrow">
+                            <span id="appliances-category-label">Appliances</span>
+                            <img src="/assets/icons/down_arrow_black.svg" alt="Down Arrow">
+                        </div>
+                    </a>
+
+                    <a
+                        href="#"
+                        class="d-flex flex-column justify-content-center align-items-center text-decoration-none text-black"
+                        role="button"
+                        aria-labelledby="travel-category-label"
+                    >
+                        <img src="/assets/images/categories/travel.svg" alt="Travel">
+                        <span id="travel-category-label">Travel</span>
+                    </a>
+
+                    <a
+                        href="#"
+                        class="d-flex flex-column justify-content-center align-items-center text-decoration-none text-black"
+                        role="button"
+                        aria-labelledby="beauty-category-label"
+                    >
+                        <img src="/assets/images/categories/beauty_toys_more.svg" alt="Beauty, Toys & More">
+                        <div class="category-label-and-arrow w-100 d-flex justify-content-between align-items-center">
+                            <span id="beauty-category-label flex-grow-1">Beauty, Toys & More</span>
```

Comment:
```suggestion
                            <span id="beauty-category-label">Beauty, Toys & More</span>
```
you added bootstrap property to `span` id attribute

Reply:
Fixed it.
https://github.com/dhruvdhaduk-simform/flipkart-skeleton-ui/pull/6/commits/3248a97ecf592c64d38322a00ba4dc1441c8428a